core-plugins/pom.xml

```diff
@@ -216,6 +222,7 @@
               co.cask.hydrator.plugin.validator.*;
               org.apache.avro.mapreduce;
               parquet.avro.*;
+              ch.ethz.ssh2.*;
```

Comment:
this should not be exported, and it shouldn't have any affect on the build. what error were you seeing?